klum/templates/deployment.yaml

```diff
@@ -19,6 +19,8 @@ spec:
             secretName: klum-github-app-key-file
       {{- end }}
       serviceAccountName: klum
+      securityContext:
```

Comment:
Maybe better use `with`  instead since the values are not required

Reply:
The `securityContext` can be set to empty object as well, but sure I'll add it to be extra explicit